pkgs/by-name/ky/kyverno-chainsaw/package.nix

```
@@ -0,0 +1,37 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "kyverno-chainsaw";
+  version = "0.1.6";
```

Version [0.1.7](https://github.com/kyverno/chainsaw/releases/tag/v0.1.7) was released recently
```suggestion
  version = "0.1.7";
```

---

For kyverno nix package there [bot](https://github.com/r-ryantm) who update version it will be used in this package also ?

---

I'm uncertain whether the bot will pick this up automatically or whether additional steps are needed for it to do so.

---

The [bot FAQ](https://ryantm.github.io/nixpkgs-update/nixpkgs-maintainer-faq/#nixpkgs-maintainer-faq ) may be helpful…

---

okay i will check regularly after release